types/conflux-web/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
```

Comment:
Remove these rules from here and fix error. New packages cannot disable any rules or use custom rules.

Reply:
can't remove, when I remove , error! 

_"ERROR: 12:6   strict-export-declare-modifiers  All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md"_